scrapers/GotFilled.yml

```diff
@@ -0,0 +1,73 @@
+name: GotFilled
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - gotfilled.com/tour/videos/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - gotfilled.com/tour/models/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //*[@id="info"]/h2/text()
+      Date:
+        selector: //*[@id="info"]/div[4]/div[3]/h5/text()
+        postProcess:
+          - parseDate: Jan 2, 2006
+      Details: //*[@id="info"]/div[1]/text()
```

Comment:
Can be refactored to `//*[@id="info"]/div[@class="desc"]/text()`

Reply:
yeah also safer